.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Feature request
+about: Suggest an idea for this project
+title: ''
```

Comment:
Is there a reason why not adding '[FEATURE]' title prefix like for the bug report ?

Reply:
No, actually this PR is only an idea, not fix set

Reply:
Introduced with https://github.com/web-vision/wv_deepltranslate/pull/145/commits/6223905a78259303e1822b8ac8b014b660fb42d4